### ISSUE TITLE
Wavefront-Proxy service with non ClusterIP type

### ISSUE THREAD

**Issue by vavdoshka** [wavefrontHQ/helm#129]
We want to leverage Wavefront-Proxy in K8s to accept and process the metrics from non-K8s cluster origin, which would require the Proxy Service to be the type of `LoadBalancer`. Which is currently not possible:
https://github.com/wavefrontHQ/helm/blob/85f8f35b3827c9a903c4e2bec5389edad2c2b529/wavefront/templates/proxy-service.yaml#L65

In addition to this, we would want to have only one proxy Deployment per cluster and use it for both internal K8s metrics traffic and the traffic with different origins external to the K8s cluster. So that means we would need to have two services with different types, one `ClusterIP` for internal traffic (as is the case now) and one with `LoadBalancer` type to handle external traffic.

So to sum up I propose to:
* add the possibility to define (enable) an extra service object for Wavefront Proxy with type LoadBalancer.

**Comment by helen-shao** [comment]
This issue is tracked with K8SSAAS-1079.
